pace.coffee

```diff
@@ -202,7 +202,7 @@ class Bar
       @el = document.createElement 'div'
       @el.className = "pace pace-active"
 
-      document.body.className = document.body.className.replace /pace-done/g, ''
+      document.body.className = document.body.className.replace / pace-done/g, ''
```

Comment:
> maybe move it into a generic removeClass fn?

I think that would be good, would you prefer string->array->string manipulation or the regex?